docs/source/trainer.rst

```diff
@@ -666,9 +666,9 @@ Under the hood the pseudocode looks like this when running *fast_dev_run* with a
 .. note::
 
     This argument is a bit different from ``limit_train/val/test_batches``. Setting this argument will
-    disable tuner, logger callbacks like ``LearningRateLogger`` and runs for only 1 epoch. This must be
-    used only for debugging purposes. ``limit_train/val/test_batches`` only limits the number of batches and won't
-    disable anything.
+    disable tuner, checkpoint callbacks, early stopping callbacks, loggers and logger callbacks like
+    ``LearningRateLogger`` and runs for only 1 epoch. This must be used only for debugging purposes.
```

Comment:
```suggestion
    ``LearningRateLogger`` and runs for only one step during the first epoch. This must be used only for debugging purposes.
```

(line will be too long, do not directly commit from suggestion)

Reply:
it can run multiple steps during an epoch if you set `fast_dev_run = int( > 1)`. Also it runs for 1 epoch only.